modules/installation-aws-config-yaml.adoc

```diff
@@ -71,7 +71,7 @@ controlPlane: <3> <4>
   name: master
   platform:
     aws:
-      lbType: NLB <12>
+      lbType: NLB
```

Comment:
```suggestion
      ifndef::openshift-origin[]
      lbType: NLB
      endif::openshift-origin[]
```
IIRC you had mentioned that this applies to enterprise only. Wrapping the parameter in this condition will prevent it from appearing in the OKD doc. While Netlify only builds enterprise, if you build locally, you can preview OKD doc. Happy to show you where if you are unable to find it.

Reply:
@mjpytlak Yes, please walk me through the preview process.